setup.py

```diff
@@ -62,6 +62,7 @@
     'nbconvert[execute]>=4.1.0',
     'nbformat>=4.0.1',
     'tornado>=4.1,<=5.1.1',  # required by nbconvert -> jupyter-client
+    'invenio-db>=1.0.13'
```

Comment:
You'll need to also add ``[versioning]`` and remove it from the ``tests_requires``